x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select.tsx

```diff
@@ -9,17 +9,21 @@ import React from 'react';
 
 import { useActions, useValues } from 'kea';
 
-import { EuiSelectable, useIsWithinMaxBreakpoint } from '@elastic/eui';
+import { EuiSelectable, EuiSelectableOption, useIsWithinMaxBreakpoint } from '@elastic/eui';
 
 import { MlModel } from '../../../../../../../common/types/ml';
 import { IndexNameLogic } from '../../index_name_logic';
 import { IndexViewLogic } from '../../index_view_logic';
 
 import { MLInferenceLogic } from './ml_inference_logic';
 import { ModelSelectLogic } from './model_select_logic';
-import { ModelSelectOption, ModelSelectOptionProps } from './model_select_option';
+import { ModelSelectOption } from './model_select_option';
 import { normalizeModelName } from './utils';
 
+type EuiSelectableOptionWithMlModelData = EuiSelectableOption & {
```

Comment:
We use this data type here because `EuiSelectableOption<MlModel>` adds all of `MlModel`'s props to the root of the type, creating the DOM attribute warnings. Nesting `MlModel` under `data` fixes the warnings because `EuiSelectable` omits anything under `data` from the rendered DOM element.